src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/validation/ValidationCode.kt

```diff
@@ -25,6 +25,17 @@ enum class ValidationCode(val message: String, val validationType: ValidationTyp
   EDS18_EDS21_EDSU18_SENTENCE_TYPE_INCORRECT("The sentence type for court case %s count %s is invalid for the sentence date entered."),
   EDS_LICENCE_TERM_LESS_THAN_ONE_YEAR("Court case %s count %s must have a licence term of at least one year."),
   EDS_LICENCE_TERM_MORE_THAN_EIGHT_YEARS("Court case %s count %s must have a licence term that does not exceed 8 years."),
+  FTR_SENTENCES_CONFLICT_WITH_EACH_OTHER("This calculation must have either a 14 or a 28 day fixed term recall sentence type."),
+  FTR_TYPE_14_DAYS_BUT_LENGTH_IS_28("You cannot have 28 days recall for a 14 day fixed term recall sentence type."),
+  FTR_TYPE_28_DAYS_BUT_LENGTH_IS_14("You cannot have 14 days recall for a 28 day fixed term recall sentence type."),
+  FTR_14_DAYS_SENTENCE_GE_12_MONTHS("The sentence length is 12 months or more, so the fixed term recall should be 28 days."),
+  FTR_14_DAYS_AGGREGATE_GE_12_MONTHS("The aggregate sentence length for the consecutive sentences is 12 months or more, so the fixed term recall should be 28 days."),
+  FTR_28_DAYS_SENTENCE_LT_12_MONTHS("The sentence is less than 12 months so the fixed term recall should be 14 days."),
+  FTR_28_DAYS_AGGREGATE_LT_12_MONTHS("The aggregate sentence length for the consecutive sentences is less than 12 months, so the fixed term recall should be 14 days."),
+  FTR_TYPE_14_DAYS_SENTENCE_GE_12_MONTHS("The sentence length is 12 months or more, so the fixed term sentence type should be 28 days."),
+  FTR_TYPE_14_DAYS_AGGREGATE_GE_12_MONTHS("The aggregate sentence length for the consecutive sentences is 12 months or more, so the fixed term sentence type should be 28 days."),
+//  FTR_28_DAYS_SENTENCE_LT_12_MONTHS("The sentence length is less than 12 months, so the fixed term sentence type should be 14 days."),
```

Comment:
Still WIP, think these extra medssages confict with another rule. Going to check with Cherky (next PR will address)